File: solutions/day-02/day02Jokot.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t): return False
        charSet = set(s)

        for c in charSet:
            if (s.count(c) != t.count(c)):
                return False

        return True

File: solutions/day-02/day02Jokot.kt
class Solution {
    fun isAnagram(s: String, t: String): Boolean {
        if (s.length != t.length) return false
        val charCountArray = IntArray(26) { 0 }

        for (c in s.toCharArray()) {
            charCountArray[c - 'a']++
        }

        for (c in t.toCharArray()) {
            charCountArray[c - 'a']--
        }

        for (value in charCountArray) {
            if (value != 0) return false
        }

        return true
    }
}
